doc/l1.md

```diff
@@ -72,3 +72,41 @@ end
 
 ```
 
+## Native pflang compilation
+
+```
+7fbe1e3d2000  4883FE16          cmp rsi, +0x16
+7fbe1e3d2004  0F8C59000000      jl 0x7fbe1e3d2063
+7fbe1e3d200a  0FB7770C          movzx esi, word [rdi+0xc]
+7fbe1e3d200e  66C1CE08          ror si, 0x08
+7fbe1e3d2012  480FB7F6          movzx rsi, si
+7fbe1e3d2016  4881FEDC050000    cmp rsi, 0x5dc
+7fbe1e3d201d  7F44              jg 0x7fbe1e3d2063
+7fbe1e3d201f  0FB7770E          movzx esi, word [rdi+0xe]
+7fbe1e3d2023  4881FEFEFE0000    cmp rsi, 0xfefe
+7fbe1e3d202a  7537              jnz 0x7fbe1e3d2063
+7fbe1e3d202c  0FB67711          movzx esi, byte [rdi+0x11]
+7fbe1e3d2030  4881FE83000000    cmp rsi, 0x83
+7fbe1e3d2037  752A              jnz 0x7fbe1e3d2063
+7fbe1e3d2039  0FB67715          movzx esi, byte [rdi+0x15]
+7fbe1e3d203d  4883FE0F          cmp rsi, +0x0f
+7fbe1e3d2041  7502              jnz 0x7fbe1e3d2045
```

Comment:
Looks like you can tighten up this code -- insead of jnz + jmp you can jz + fallthrough, no?